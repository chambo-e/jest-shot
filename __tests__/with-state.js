import shot from '../src/';
import WithState from '../__components__/with-state';

const mutations = [
	{
		description: 'without state, should snap default state',
	},
	{
		description: 'with custom state',
		state: {
			snapPhrase: 'test snapshot 1',
		},
	},
	{
		description: 'with custom state bis',
		state: {
			snapPhrase: 'should be snapshoted',
		},
	},
];

shot(WithState, mutations);
