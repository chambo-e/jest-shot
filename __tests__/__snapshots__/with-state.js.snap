// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentWithState with custom state 1`] = `
<div>
  test snapshot 1
</div>
`;

exports[`ComponentWithState with custom state bis 1`] = `
<div>
  should be snapshoted
</div>
`;

exports[`ComponentWithState without state, should snap default state 1`] = `
<div>
  Should be snapshoted only when no setState
</div>
`;
